src/main/java/com/vmware/vipclient/i18n/l2/service/NumberFormatService.java

```diff
@@ -114,7 +119,9 @@ public String format(Object value, String currencyCode, Integer fractionSize, Lo
         }
         NumberFormat numberFormat = NumberFormat.getInstance(numberFormatData, style);
         formatNumber = numberFormat.format(value, fractionSize);
-        if (style == NumberFormat.PERCENTSTYLE) {
+        if (style == NumberFormat.INTEGERSTYLE) {
+            //TODO
```

Comment:
what needs to do here? add more details?

Reply:
Currently for number format only support 'decimal&percent&currency' format, doesn't support 'integer' format, so it's need to do in future.